src/components/KPrompt/KPrompt.vue

```diff
@@ -227,62 +226,92 @@ onBeforeUnmount(() => {
+
+      .k-prompt-header-content {
+        align-items: center !important;
+        display: flex !important;
+        width: 100% !important;
+
+        .warning-icon {
+          color: $tmp-color-yellow-400;
+          margin-right: var(--kui-space-40, $kui-space-40) !important;
+        }
+
+        .close-button {
+          margin-left: var(--kui-space-auto, $kui-space-auto);
+        }
+      }
     }
 
     .divider {
       border: none;
-      border-top: 1px solid var(--grey-300);
+      border-top: var(--kui-border-width-10, $kui-border-width-10) solid var(--grey-300, var(--kui-color-border-neutral-weak, $kui-color-border-neutral-weak));
       /* subtract parents padding from margin to take full width of modal */
-      /* use interpolation for the var in calc to not break postcss */
-      margin: 16px calc(#{var(--spacing-lg)} * -1) 0;
+      margin: var(--kui-space-60, $kui-space-60) calc($kPromptModalPadding * -1) var(--kui-space-0, $kui-space-0);
     }
 
     .k-modal-content {
       .k-modal-header.modal-header {
         display: flex;
-        padding-bottom: var(--spacing-xs);
+        padding-bottom: var(--spacing-xs, var(--kui-space-40, $kui-space-40));
         width: 100%;
 
         .close-button .k-button {
-          margin-top: -8px;
-          padding: var(--spacing-xs);
+          @include non-visual-button;
+          margin-top: calc(-1 * var(--kui-space-40, $kui-space-40));
+          padding: var(--spacing-xs, var(--kui-space-40, $kui-space-40));
         }
       }
 
       .k-modal-body.modal-body {
         width: 100%;
 
-        .k-prompt-body .k-prompt-body-content {
-          color: var(--grey-600);
-          font-size: var(--type-md);
-          line-height: 24px;
-          max-height: var(--KPromptMaxHeight, 300px);
-          overflow-x: hidden;
-          overflow-y: auto;
-          padding-bottom: var(--spacing-md);
-          text-align: start;
-          white-space: normal; // in case inside KTable
-          width: 99%;
-
-          @media screen and (min-width: $viewport-md) {
-            max-height: var(--KPromptMaxHeight, 500px);
-          }
+        .k-prompt-body {
+          width: 100% !important;
 
-          .k-prompt-confirm-text {
-            margin-top: var(--spacing-lg);
-            .k-input {
-              width: 100%;
+          .k-prompt-body-content {
+            color: var(--grey-600, var(--kui-color-text-neutral-strong, $kui-color-text-neutral-strong));
+            font-size: var(--type-md, var(--kui-font-size-40, $kui-font-size-40));
+            line-height: var(--kui-line-height-40, $kui-line-height-40);
+            max-height: var(--KPromptMaxHeight, 300px);
+            overflow-x: hidden;
+            overflow-y: auto;
+            padding-bottom: var(--spacing-md, var(--kui-space-60, $kui-space-60));
+            text-align: start;
+            white-space: normal; // in case inside KTable
+            width: 100% !important;
```

Comment:
question: Why did the width change from `99%` to `100%`?

Reply:
We had `.w-100` utility class on this element which was clearly superseding this width rule.